src/states/client.rs

```diff
@@ -388,18 +388,8 @@ impl Bootstrapped for Client {
 
 #[cfg(feature = "use-mock-crust")]
 impl Client {
-    /// Resends all unacknowledged messages.
-    pub fn resend_unacknowledged(&mut self) -> bool {
-        let timer_tokens = self.ack_mgr.timer_tokens();
-        for timer_token in &timer_tokens {
-            self.resend_unacknowledged_timed_out_msgs(*timer_token);
-        }
-        !timer_tokens.is_empty()
-    }
-
-    /// Are there any unacknowledged messages?
-    pub fn has_unacknowledged(&self) -> bool {
-        self.ack_mgr.has_pending()
+    pub fn poll(&mut self) -> Vec<u64> {
```

Comment:
pls see the the review at https://github.com/maidsafe/routing/pull/1438/files#diff-c9f7984a0a4f0ccaf63472cce67a9e67R3152